src/components/SearchBar/Searchbar.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import './searchbar.css';
+
+const Searchbar = props => {
+    return(
+        <>
+            <input className={"searchbar"} type="text" placeholder="&#128269; Search"></input>
```

Comment:
Did you mean to write `className={"searchbar"}` instead of `className="searchbar"`